src/web/components/StickyBottomBanner/StickyBottomBanner.tsx

```diff
@@ -149,36 +105,41 @@ export const StickyBottomBanner = ({
         );
     }, []);
 
-    useEffect(() => {
-        // Don't do anything until isSignedIn is defined as we only want to set
-        // asyncBrazeUuid once
-        if (isSignedIn === undefined) {
-            return;
-        }
-
+    useOnce(() => {
         if (isSignedIn) {
             setAsyncBrazeUuid(getBrazeUuid(idApiUrl));
         } else {
             setAsyncBrazeUuid(Promise.resolve(null));
         }
     }, [isSignedIn, idApiUrl]);
 
-    if (
-        isSignedIn === undefined ||
-        asyncCountryCode === undefined ||
-        asyncBrazeUuid === undefined ||
-        shouldHideSupportMessaging === undefined
-    ) {
-        return null;
+    useOnce(() => {
+        const CMP = buildCmpBannerConfig();
+        const readerRevenue = buildReaderRevenueBannerConfig(
+            CAPI,
+            isSignedIn || false, // TODO: Remove this
+            asyncCountryCode || Promise.resolve('GB'), // TODO: Remove this
```

Comment:
RE the TODO's - removal before or after this PR?

Reply:
Based on some tests and discussions outisde of this PR, I think these fallbacks might need to remain to keep typescript happy. I wasn't able to find a way to tell TS that these variables are certain to be defined

Reply:
So they should be removed as part of this PR (assuming we can't think of a better solution)

Reply:
As discussed offline would be great to figure out a way of not needing these defaults, but keeping them for the purposes of moving forward with this sounds good.